src/AppMarkVoterCardInvalid.test.tsx

```diff
@@ -146,7 +146,7 @@ describe('Mark Card Void when voter is idle too long', () => {
 
     // 30 seconds passes, Expect voided card
     advanceTimers()
-    await wait()
+    await wait() // TODO: unsure why this `wait` is needed, but it is.
```

Comment:
I wonder if the double state change is part of this, thus the need to call `wait()` which calls `act()`. We should re-try this after my PR lands.

Reply:
Agreed.